File: KingdomCodeTool/wwwroot/Download/202501201119/Angular/shared/Sys_Menus.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_Menus } from './Sys_Menus.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_MenusService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Menu_Id', 'Menu_Code', 'Menu_Name', 'Language_Id', 'Menu_Level', 'Idx', 'Parent_Id', 'Function_Id', 'Function_Argument', 'Default_Right', 'Enabled', 'MenuBar', 'MenuLeft', 'Is_System', 'Is_Report', 'Report_Id', 'Shortcut_Key', 'Shortcut_Character', 'WindowState', 'ShowDialog', 'Creation_Date', 'Created_By', 'Last_Update_Date', 'Last_Updated_By', 'Function_Argument_Notes', 'Remark'];

    List: Sys_Menus[] | undefined;
    ListFilter: Sys_Menus[] | undefined;
    FormData!: Sys_Menus;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_Menus";
    }
}

File: KingdomCodeTool/wwwroot/Download/202501201119/Service/Implement/Sys_MenusService.cs
namespace Service_eHospital_DongNai_A_System.Implement
{
    public class Sys_MenusService : BaseService<Sys_Menus, ISys_MenusRepository>
    , ISys_MenusService
    {
    private readonly ISys_MenusRepository _Sys_MenusRepository;
    public Sys_MenusService(ISys_MenusRepository Sys_MenusRepository) : base(Sys_MenusRepository)
    {
    _Sys_MenusRepository = Sys_MenusRepository;
    }
    }
    }

